kover-gradle-plugin/docs/migrations/migration-to-0.8.0.md

```diff
@@ -124,7 +124,7 @@ If you use the extensions property to access the extension
 The `0.7.x` version used two project extensions named `kover` and `koverReports`.
 This was confusing and made it difficult to search through documentation or examples.
 
-Since `0.8.0`, there is only one `kover` extension left.
+Since `0.8.1`, there is only one `kover` extension left.
```

Comment:
Are these changes meant to be here? Shouldn't it remain `0.8.0`?
They happened multiple times in this file

Reply:
In this case, yes, it is worth leaving `0.8.0`.

For this case, we can rephrase it like: in any version `0.8.x` there is only one `kover` extension left.

Reply:
Fixed, thanks!